Source/WebCore/css/typedom/transform/CSSScale.cpp

```diff
@@ -63,7 +63,7 @@ ExceptionOr<Ref<CSSScale>> CSSScale::create(CSSNumberish x, CSSNumberish y, std:
 
 ExceptionOr<Ref<CSSScale>> CSSScale::create(CSSFunctionValue& cssFunctionValue)
 {
-    auto makeScale = [&](const Function<ExceptionOr<Ref<CSSScale>>(Vector<RefPtr<CSSNumericValue>>&&)>& create) -> ExceptionOr<Ref<CSSScale>> {
+    auto makeScale = [&](const Function<ExceptionOr<Ref<CSSScale>>(Vector<RefPtr<CSSNumericValue>>&&)>& create, size_t minNumberOfComponents, std::optional<size_t> maxNumberOfComponents = std::nullopt) -> ExceptionOr<Ref<CSSScale>> {
```

Comment:
Can this be size_t maxNumberOfComponents = minNumberOfComponents instead?  That would slightly simplify the code below.

Reply:
I had the same idea, but the compiler disagreed:

```
css/typedom/transform/CSSScale.cpp:66:174: error: default argument references parameter 'minNumberOfComponents'
    auto makeScale = [&](const Function<ExceptionOr<Ref<CSSScale>>(Vector<RefPtr<CSSNumericValue>>&&)>& create, size_t minNumberOfComponents, size_t maxNumberOfComponents = minNumberOfComponents) -> ExceptionOr<Ref<CSSScale>> {
```